homeassistant/components/person/__init__.py

```diff
@@ -80,6 +81,8 @@ def __init__(self, hass: HomeAssistantType, component: EntityComponent,
 
         config_data = self.config_data = OrderedDict()
         for conf in config_persons:
+            if not conf:
```

Comment:
This is weird. I think that this is a flaw in packages and should not be fixed in the person component. That means that any other integration in the future will need to add checks in case packages are used, that's wrong.

Reply:
An alternative is that `ensure_list` should return `[]` for an empty config branch.

It’s supposed to be null, but with our Yaml loader we convert an empty branch to an empty OrderedDict

Reply:
Added a `remove_falsy' validator to fix this. Also rolled back changes to the tests to show it has no impact